common/core/src/commonMain/kotlin/com/trafi/ui/theme/internal/type/LightDarkColor.kt

```diff
@@ -0,0 +1,5 @@
+package com.trafi.ui.theme.internal.type
+
+class LightDarkColor(val light: Color, val dark: Color)
+
+internal infix fun Color.dark(dark: Color) = LightDarkColor(light = this, dark = dark)
```

Comment:
This is the magic line that enables us to write `someColor = White dark Black`? 😅
Nice 👍

Reply:
Yes, indeed 🙂 The "magic" part is the `infix` modifier - it allows to call the function using infix notation, omitting the dot and parentheses.

If interested, you can [read a few more details in Kotlin's official documentation](https://kotlinlang.org/docs/reference/functions.html#infix-notation).